README.md

```diff
@@ -1,67 +1,46 @@
-Apertium Html-tools
-====================
-
-[![CircleCI](https://circleci.com/gh/apertium/apertium-html-tools.svg?style=svg)](https://circleci.com/gh/apertium/apertium-html-tools)
-
-[Apertium Html-tools][1] is a web application providing a fully localised
-interface for text/document/website translation, analysis, and generation
-powered by [Apertium][2]. Designed with a dead simple build process, minimal
-dependencies and only static resources for quick and easy deployment with
-any web server, it is relatively lightweight and user-friendly. Html-tools
-relies on an Apertium HTTP API such as [Apertium-apy][3] or [ScaleMT][4]
-(to a lesser extent).
-
-More information along with instructions for localization is available on the
-[Apertium Wiki][5].
-
-Prerequisites
--------------
-
-* Python 3
-* curl
-
-Setup
------
-
-1. Copy `config.conf.example` to `config.conf` and edit it.
-1. Then type `make`.
-
-Running
--------
-
-The static files within `/build` (generated by `make`) can be served by any
-server. You can use whatever you like, including Apache or even just Python's
-HTTP server. For example, to run on locally on `http://localhost:8082` you
-can do the following.
-
-    python3 -m http.server 8082
-
-A completely local setup with language data and translation from Apertium,
-an APy instance, and Nginx to serve Html-tools is provided via [Docker][6].
-After installing Docker, point your `APY_URL` in `config.conf` to
-`http://localhost:2737`, run the following, and check `http://localhost:8082`.
-Any changes to the local version of Html-tools will trigger a `make`.
-
-    docker-compose up
-
-For production usage, remember to
-
-1. Build with `jsmin`, `csscompressor` and `htmlmin` using
-   `pip install -r requirements-prod.txt`.
-1. Enable gzip compression on your server.
-
-Contributing
-------------
-
-Run `pip install -r requirements-dev.txt` to get development dependencies.
-
-[CircleCI][7] runs tests and generates build artifacts including
-a live copy of Html-tools using `config.conf.example`.
-
-[1]: http://wiki.apertium.org/wiki/Apertium-html-tools
-[2]: http://apertium.org
-[3]: http://wiki.apertium.org/wiki/Apertium-apy
-[4]: http://wiki.apertium.org/wiki/ScaleMT
-[5]: http://wiki.apertium.org/wiki/Apertium-html-tools
-[6]: https://www.docker.com/
-[7]: https://circleci.com/
+Localization
```

Comment:
This change is to the incorrect README?

Reply:
Really, I got lost. I don't understand anything. I can't imagine what is happening. I'm uploading the files where I get something in "git diff", and in this particular file I see changes only in 3 lines. I better close everything, and send to you the two files with the strings by email. We are both wasting a lot of time in something that seems to be (and should be) extremely easy and mechanical.

Reply:
>  I'm uploading the files

This is the crux of the issue. The recommended workflow is to push your branch to GitHub via `git push` and then GitHub will provide a link to create a pull request.

Reply:
It does not seem GitHub provides any link to me :

```
$ git add frp.json 

$ git commit README.md fra.json frp.json -m "+frp & actualisation fra"
[master 2054055] +frp & actualisation fra
 3 files changed, 135 insertions(+), 68 deletions(-)
 rewrite assets/strings/fra.json (61%)
 create mode 100644 assets/strings/frp.json

$ git push
Username for 'https://github.com': hectoralos
Password for 'https://hectoralos@github.com': 
Comptant els objectes: 7, fet.
Delta compression using up to 8 threads.
Comprimint objectes: 100% (6/6), fet.
Escrivint els objectes: 100% (7/7), 3.86 KiB | 0 bytes/s, fet.
Total 7 (delta 5), reused 2 (delta 1)
remote: Resolving deltas: 100% (5/5), completed with 5 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: You're not authorized to push to this branch. Visit https://docs.github.com/articles/about-protected-branches/ for more information.
To https://github.com/apertium/apertium-html-tools/
 ! [remote rejected] master -> master (protected branch hook declined)
error: s'ha fallat en pujar algunes referències a 'https://github.com/apertium/apertium-html-tools/'
```